pyproject.toml

```diff
@@ -1,64 +1,57 @@
 #:schema https://json.schemastore.org/pyproject.json
 
-[tool.poetry]
-authors = ["ONEKEY <support@onekey.com>"]
+[project]
+authors = [{ name = "ONEKEY", email = "support@onekey.com" }]
+dependencies = [
+  "arpy>=2.3.0",
+  "attrs>=23.1.0",
+  "click>=8.1.7",
+  "cryptography>=41.0",
+  "dissect-cstruct>=4.0,<5.0",
+  "jefferson>=0.4.5",
+  "lark>=1.1.8",
+  "lief>=0.16.1",
+  "lz4>=4.3.2",
+  "plotext>=4.2.0,<6.0",
+  "pluggy>=1.3.0",
+  "pyfatfs>=1.0.5",
+  "pyperscan>=0.3.0",
+  "python-magic>=0.4.27",
+  "rarfile>=4.1",
+  "rich>=13.3.5",
+  "structlog>=24.1.0",
+  "treelib>=1.7.0",
+  "ubi-reader>=0.8.9",
+  "unblob-native>=0.1.5",
+]
 description = "Extract files from any kind of container formats"
-license = "MIT"
```

Comment:
Was this removed on purpose? The schema linked at the top of the file (https://json.schemastore.org/pyproject.json) allows `license` field here.

Reply:
This a draft for a reason, although the license key is for custom license
https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license

Reply:
> although the license key is for custom license

the doc you linked mentions SPDX explicitly

Reply:
I have zero idea, where I've read it or just dreamt about it, but it is cool then